README.md

```diff
@@ -1,218 +1,80 @@
-# Cloudant Spring Library
+# Cloudant Spring Boot Starter
 
-The is a library to provide Spring developers easy configuration of the [official Cloudant library for Java](https://github.com/cloudant/java-cloudant/).
+This is a rewrite of the "spring plugin", to provide a "starter project" which allows users to get up and running quickly,
+as well as providing autoconfiguration via application.properties or another configuration source (environment, java properties, etc)
 
-The library is split into two parts:
-* `cloudant-spring-boot-starter` for [Spring Boot](https://projects.spring.io/spring-boot/) applications
+## Using
 
-* `cloudant-spring-framework` for [Spring Framework](https://projects.spring.io/spring-framework/) applications
+Here is a very quick run-through of what a project using this starter would look like.
 
-
-* [Installation and Usage](#installation-and-usage)
-* [Getting Started](#getting-started)
-* [Related Documentation](#related-documentation)
-* [Development](#development)
-    * [Contributing](CONTRIBUTING.md)
-    * [Test Suite](CONTRIBUTING.md#running-the-tests)
-    * [Using in Other Projects](#using-in-other-projects)
-    * [License](#license)
-    * [Issues](#issues)
-
-## Installation and Usage
-
-### Spring Boot Applications
+In this example, we are writing a web app which provides a REST endpoint, `/all_dbs` which returns a list of all
+databases for the Cloudant account.
 
 Gradle:
 ```groovy
 dependencies {
-    compile group: 'com.cloudant', name: 'cloudant-spring-boot-starter', version: '0.0.4'
+  implementation 'org.springframework.boot:spring-boot-starter-web'
+  implementation 'com.cloudant:cloudant-spring-boot-starter:0.0.5-SNAPSHOT'
 }
 ```
 
-Maven:
-~~~ xml
-<dependency>
-  <groupId>com.cloudant</groupId>
-  <artifactId>cloudant-spring-boot-starter</artifactId>
-  <version>0.0.4</version>
-</dependency>
-~~~
-
-### Spring Framework Applications
-
-Gradle:
-```groovy
-dependencies {
-    compile group: 'com.cloudant', name: 'cloudant-spring-framework', version: '0.0.4'
-}
+application.properties:
+```properties
+cloudant.create-db=true
+cloudant.db=my-db
```

Comment:
While it is useful for general configuration, these two properties don't really make sense for the chosen example (listing all DBs).
```suggestion
```